test/integration/update.reservation.test.js

```diff
@@ -0,0 +1,66 @@
+process.env.NODE_ENV = 'test';
```

Comment:
You can add these env variable berore `npm run ..` instead here, 
on linux and mac: `NODE_ENV=test DEBUG=knex* npm run ..`